src/functions/pollJournals/framework/handler.ts

```diff
@@ -0,0 +1,68 @@
+import { APIGatewayProxyEvent, Context } from 'aws-lambda';
+import createResponse from '../../../common/application/utils/createResponse';
+import Response from '../../../common/application/api/Response';
+import { getExaminers } from './repo/mysql/examiner-repository';
+import { getTestSlots } from './repo/mysql/test-slot-repository';
+import * as mysql from 'mysql';
+import { chunk } from 'lodash';
+import { getNonTestActivities } from './repo/mysql/non-test-activity-repository';
+import { getPersonalCommitments } from './repo/mysql/personal-commitment-repository';
+import { getAdvanceTestSlots } from './repo/mysql/advance-test-slots-repository';
+import { getDeployments } from './repo/mysql/deployment-repository';
+import { transform } from '../application/transformer';
+import { ExaminerAdvanceTestSlot } from '../domain/examiner-advance-test-slot';
+import { ExaminerDeployment } from '../domain/examiner-deployment';
+import { ExaminerNonTestActivity } from '../domain/examiner-non-test-activity';
+import { ExaminerPersonalCommitment } from '../domain/examiner-personal-commitment';
+import { ExaminerTestSlot } from '../domain/examiner-test-slot';
+import { JournalWrapper } from '../domain/journal-wrapper';
+import { saveJournals } from './repo/dynamodb/journal-repository';
+
+export interface AllDatasets {
+  testSlots: ExaminerTestSlot[];
+  personalCommitments: ExaminerPersonalCommitment[];
+  nonTestActivities: ExaminerNonTestActivity[];
+  advanceTestSlots: ExaminerAdvanceTestSlot[];
+  deployments: ExaminerDeployment[];
+}
+
+export async function handler(event: APIGatewayProxyEvent, fnCtx: Context): Promise<Response> {
+  const examinerBatchSize = Number.parseInt(process.env.EXAMINER_BATCH_SIZE || '250', 10);
+  const pool: mysql.Pool = mysql.createPool({
+    host: process.env.MYSQL_HOST,
+    database: process.env.MYSQL_DATABASE,
+    user: process.env.MYSQL_USER,
+    password: process.env.MYSQL_PASSWORD,
+    connectionLimit: 50,
+  });
+
```

Comment:
process.env. variables need to be retrieved more securely in the future. (encrypted or pulled form secure vault)

Reply:
Added a config facade + Secrets Manager support.